src/server.c

```diff
@@ -800,6 +800,10 @@ struct redisCommand redisCommandTable[] = {
      "read-only fast random @keyspace",
      0,NULL,1,1,1,0,0,0},
 
+    {"expiretime",expiretimeCommand,3,
```

Comment:
Not sure if it is a good idea to expose this command as a new redis command. I think most redis users would prefer to see expire time as a relative value instead of the absolute epoch time value. 

If this command is used for testing purposes only, can you make it into a sub-command for `DEBUG` command?

Reply:
I'm introducing it for testing purposes. But I do think it has use cases outside testing. It allows users to understand the relative distance between the expiration deadlines of different keys. E.g. if two keys would be expired together. Today there is no way to do that. I think this family of use cases deserve a top level command, assuming we don't only introduce top level commands for super popular use cases.

I preferred introducing `EXPIRETIME` over a `DEBUG` sub-command also because I wanted to colocate the implementation of `expiretimeCommand` in `expire.c` next to `ttlCommand` so I can easily re-use `ttlGenericCommand`.

Reply:
Remember, setting expiry with absolute timestamp is not a new feature. People can already do that today. I am not sure if this change justifies adding a new user facing command. 

I still prefer us to handle the expiry replication as an internal mechanism inside redis that is done transparently from the user perspective.

Reply:
Yes people can set expiry with absolute timestamps today. But they can't query them. Do you agree that it can be useful to query absolute timestamps outside of testing?

Reply:
i'm in favor of adding that command (not as debug command).
Someone wanting to use DUMP and RESTORE, may want to use it instead of PTTL (to provide ABSTTL to RESTORE)